cfpb/cfgov-refresh
# Whitelisting GTM debugger scripts

Whitelisting GTM debugger scripts


## Changes

- Modified `cfgov/cfgov/settings/base.py` to whitelists scripts needed for the GTM debugger. 

## Testing

1. Visit Google Tag Manager and enable Preview mode.
2. Visit `localhost:8000`. 
3. Inspect the page and notice the GTM debugger is visible.

## Notes

- There are other font, images, and styles I didn't whitelist because they aren't crucial to the page functioning.

## Todos

-

## Checklist
